### ISSUE TITLE
Black screen is appearing while app starting in ionic 3?

### ISSUE THREAD

**Issue by sudhan-ionic** [ionic-team/ionic#13281]
Like this Black screen is appearing while app starting in ionic 3

![screenshot_2017-10-20-17-57-13_720-1](https://user-images.githubusercontent.com/33175538/32158861-d95d908a-bd70-11e7-8817-46a0b40c5ef9.png)

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us! Would you be able to provide a sample application via GitHub that demonstrates the issue you are having? Thanks for using Ionic!

**Comment by sudhan-ionic** [comment]
![videotogif_2017 10 30_16 52 19](https://user-images.githubusercontent.com/33175538/32168493-d7afafc6-bd92-11e7-8293-630bc9ff8a80.gif)

**Comment by kensodemann** [comment]
Thank you for the reply. I will need a sample application via GitHub to help you out, however. The screenshots don't really help all that much.

Does this happen when you just fire up an application created from one of our starter application without any modification?

**Comment by sudhan-ionic** [comment]
ok

**Comment by sudhan-ionic** [comment]
Sorry, How to fire a project via git
can you help me???

**Comment by kensodemann** [comment]
I am not sure what version control system you are using now. I assume git, but if not then learning enough git to commit your changes locally will be a good idea (if you don't know git, you should learn it, it is a good skill to have on your resume). There are several tutorials for git if you Google around. For example: https://try.github.io. The official documentation is here: https://git-scm.com

If you already know how to use git, but don't know how to use GitHub, then the best place to start is here: https://help.github.com

Basically, you will do this:

1. create a sample app (example: `ionic start show-blank-screen-bug tabs`). This will create the project with a local git repo
2. create a repo on GitHub and follow the instructions to associate your local repo with the remote repo on GitHub
3. duplicate this issue
4. commit your changes to your local repo as you go
5. when ready, push your changes from your local repo to the remote repo on GitHub

**Comment by sudhan-ionic** [comment]
here is project in github
https://github.com/sudhan-ionic/Black-screen-in-ionic

**Comment by kensodemann** [comment]
Thank you for providing that repo. That was very helpful. I believe you are talking about the blank screen that comes up before the splash screen.

Here is some more information on that: https://stackoverflow.com/questions/30342933/blank-screen-comes-before-splash

I am going to close this issue as this is not an Ionic Framework issue since this occurs before the application even launches. You can try some of the theming stuff mentioned in the SO article.

**Comment by sudhan-ionic** [comment]
I slove my issues 
Thanks kensodemann

**Comment by sudhan-ionic** [comment]
Here I have attached my solution I hope it will helpful to you.
https://techionichybride.blogspot.in/2017/11/how-to-reduce-black-screen-while-app_8.html

**Comment by selotoprak** [comment]
IonicModule.forRoot(MyApp, {
      preloadModules: true
    })

Do that and thank me later :)

**Comment by sudhan-ionic** [comment]
where I want to place this code @selotoprak

**Comment by selotoprak** [comment]
app.module.ts file

**Comment by sudhan-ionic** [comment]
ok fine i will do it

**Comment by selotoprak** [comment]
This issue is only exist when using lazy loading.

**Comment by sudhan-ionic** [comment]
![image](https://user-images.githubusercontent.com/33175538/34190948-a6b65ea2-e569-11e7-925a-c4f2d820bc09.png)
Like this Correct

**Comment by selotoprak** [comment]
[cid:08BB729A-0A1E-464C-A5B2-525780A6F7A1]

On Dec 19, 2017, at 8:10 PM, sudhan-ionic <notifications@github.com<mailto:notifications@github.com>> wrote:


[image]<https://user-images.githubusercontent.com/33175538/34190948-a6b65ea2-e569-11e7-925a-c4f2d820bc09.png>
Like this Correct

**Comment by sudhan-ionic** [comment]
Like this correct @selotoprak 

![image](https://user-images.githubusercontent.com/33175538/34191108-868d206a-e56a-11e7-9e35-17b86211d3ca.png)

**Comment by AhmedHamedTN** [comment]
@sudhan-ionic  no, it is already declared in your imports above ... under BrowserModule
